tests/integration/sql/test_sql.py

```diff
@@ -5,37 +5,52 @@
 
 import pandas as pd
 import pytest
+import sqlalchemy
 
 import daft
 from daft.context import set_execution_config
 from tests.conftest import assert_df_equals
 from tests.integration.sql.conftest import TEST_TABLE_NAME
 
 
+@pytest.fixture(scope="session", params=["url", "conn"])
+def db_conn(request, test_db):
```

Comment:
Do we need to fully parametrize every test with url vs conn? Or can we just have one dedicated test to ensure that passing in a conn instead of a URL works as expected?

Just a little concerned that it might bloat our tests, without really giving us much more coverage since we're just passing in a SQL statement in either case.

Reply:
Yeah I guess we don't, since the sqlalchemy connection path is also tested via the Trino connections as it's not supported by ConnectorX. Will remove this parametrization and add a dedicated test